apps/accounts/tests/test_views.py

```diff
@@ -225,3 +225,119 @@ def test_auth_required(self):
 
     def test_verbs_allowed(self):
         self.verbs_allowed(self.cls, ['get'])
+
+
+@override_settings(FXA_CONFIG=FXA_CONFIG)
+class TestAuthorizeView(APITestCase):
+
+    def setUp(self):
+        create_switch('fxa-auth', active=True)
+        self.url = reverse('accounts.authorize')
+        patcher = mock.patch('accounts.views.verify.fxa_identify')
+        self.fxa_identify = patcher.start()
+        self.addCleanup(patcher.stop)
```

Comment:
You have a few of those that are very similar now, do you want to create a base class that does that, and maybe uses `self.URL` as the parameter of the `reverse` line 235 above?

I'm not saying you need to, having less indirection is good too, just a thought ;)